tasks-june/script.js

```diff
@@ -0,0 +1,112 @@
+/*	TASK 1
+	Avem 2 inputuri de tip text intr-un formular care reprezinta numele si prenumele. 
+	Sa se afiseze cu ajutorul unui alert un mesaj de salut.
+*/
+function greet() {
+	alert(`Hello ${nume.value} ${prenume.value}. How are you today`);
+}
+
+
+/*	TASK 2
+	Avem un dropdown cu 3 optiuni. 
+	Cu ajutorul unui buton sa se stearga optiunea selectata din dropdown.
+*/
+function deleteOp() {
+	let toDelete = dropdown.value;
+	let allOptions = document.querySelectorAll('option');
+	for (let item of allOptions) {
```

Comment:
you can use filter on array
for example
array = [1,2,3,4]
value = 3
result = array.filter((item) => item !== value); // result = [1,2,4]